cmd/context/options.go

```diff
@@ -31,6 +31,7 @@ type ContextOptions struct {
 	Show                  bool
 	Save                  bool
 	IsCtxCommand          bool
+	FromNamespace         bool
```

Comment:
I find this tied to the cobra command and very specific and hard to reuse. What do you think about using a `DefaultNamespace string` property instead? That way it has an independent meaning from the command and can be reused.

Then in the error message we'd say "Couldn't access namespace ..., defaulting to ..."

Reply:
@AgustinRamiroDiaz This option is to know if the command is executed from the namespace command instead of the context one. It's the same as `IsCtxCommand`, `show` or `save`. 

If we use `DefaultNamespace` with the name of the personal namespace only when it's a namespace command it's not obvious and will be reused in a wrong way. 

We'd need to rethink how the `namespace` and `context` command are written but right now there is no plans to do it, so the least intrusive way to do it I think is this one. WDYT?

Reply:
hmmm yeah that's a good point. I felt like we could improve it but you are right, it'll get misused if the name is `DefaultNamespace`. I still feel that it's not a good abstraction, but to change it we'd need to rethink the whole command to have more composable pieces instead of one big command.

I still think that we should rename it since in the comment below we'd be using it in the context command too

Reply:
We could name it `checkIfHasAccessToNamespace` or something like this. WDYT?

Reply:
Yea that makes sense, let me change that around

Reply:
I used `CheckNamespaceAccess`, let me know if that works

Reply:
That's perfect